miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/NotificationDto.java

```diff
@@ -63,6 +97,15 @@ public void setLaneCount(int laneCount) {
     this.laneCount = laneCount;
   }
 
+  /**
+   * Get the status of the run.
+   * 
+   * This the current status of the run. This may be impossible to correctly detect depending on the platform. MISO will not record the
+   * completion date if the health is not done. If the status has been changed by a human, any updates provided in notifications are
```

Comment:
`if the health is not [enumeration of completed states]` or `if the health is "Running", "Unknown", or "Started"` may be more clear (or move the `@see HealthType#isDone()` to a parenthetical at the end of that sentence).